.github/workflows/CI.yml

```diff
@@ -124,14 +124,14 @@ jobs:
         env:
           OS_MIGRATIONS_GRADLE_SCAN_TOS_AGREE_AND_ENABLED: ''
       - name: Install python dependencies
-        working-directory: test
+        working-directory: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test
         run: |
           python -m pip install --upgrade pipenv
           pipenv install --deploy --dev
       - name: Run E2E test script
-        working-directory: test
+        working-directory: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test
         run: |
-          pipenv run pytest tests.py --unique_id="testindex"
+          docker exec $(docker ps --filter "name=migration-console" -q) pytest /root/lib/integ_test/integ_test/replayer_tests.py --unique_id="testindex"
```

Comment:
Can we run all tests?

Reply:
We don't currently have a mechanism for running Backfill in our docker setup. Once this is added we should update this.

Reply:
Is there a jira item, can we add one to track?  I'd love to see this merge running everything, but happy if we are working incrementally.